src/main/java/org/primefaces/renderkit/CoreRenderer.java

```diff
@@ -113,8 +110,10 @@ protected void renderPassThruAttributes(FacesContext context, UIComponent compon
         renderDynamicPassThruAttributes(context, component);
     }
 
-    protected void renderPassThruAttributes(FacesContext context, UIComponent component, List<String>... attrs) throws IOException {
+    @SafeVarargs
+    protected final void renderPassThruAttributes(FacesContext context, UIComponent component, List<String>... attrs) throws IOException {
         if (attrs == null) {
+            renderDynamicPassThruAttributes(context, component);
```

Comment:
If varargs is null the old code still called renderDynamicPassthrough so this puts that back.  Looks at ColorPickerRenderer as an example